### ISSUE TITLE
@nimpl/classnames-minifier update 3.0.0 -> 3.2.0 broke the build

### ISSUE THREAD

**Issue by dartess** [vordgi/nimpl-classnames-minifier#111]
Hi!

I updated the package (3.0.0 -> 3.2.0). 
If I delete the cache, I get 5 errors something like this:

```
./.../....module.scss
Error: ENOENT: no such file or directory, open '/.../.next/cache/ncm/ncm/0c212ed8-cfa0-4a52-a41d-453dc2303045'
    at Object.writeFileSync (node:fs:2342:20)
    at /.../node_modules/classnames-minifier/dist/lib/classnames-minifier-postloader.js:12:26
    at Array.forEach (<anonymous>)
    at Object.default_1 (/.../node_modules/classnames-minifier/dist/lib/classnames-minifier-postloader.js:10:51)
```

I use `next@14.0.4` and didn't update it.

This is reproduced both in bitbucket pipelines and in verсel, as well as locally.

I will be happy to provide additional information if required. For now I have paused the dependency update task.

**Comment by vordgi** [comment]
Hello. Sorry for this problem. Fixed in 3.2.1.

_Thank you very much for writing!_

**Comment by dartess** [comment]
it was blazing fast
thanks! seems ok now
